Game.cpp

```diff
@@ -21,6 +23,16 @@ Game::Game(uint32 sizeX, uint32 sizeY)
 	fInGame(false),
 	fScore(0)
 {
+	std::ifstream highscore("Highscore.dat", std::ios::binary);
```

Comment:
You should put Highscore.dat under B_USER_SETTINGS_DIRECTORY, if you do it like that then the file will end up at whatever current directory it is (even /system/apps, which is not good)

Example here: https://github.com/haikuports/haikuports/wiki/FindDirectory